lutris/gui/widgets/game_bar.py

```diff
@@ -62,8 +62,9 @@ def update_view(self):
         y_offset = 42
         if self.game.is_installed:
             self.put(self.get_runner_button(), x_offset, y_offset)
-            self.put(self.get_runner_label(), x_offset + 45, y_offset)
-            x_offset += 135
+            x_offset += 80
+            self.put(self.get_runner_label(), x_offset, y_offset)
+            x_offset += 95
```

Comment:
Style change: incremented the offset in two places. I think this is clearer and probably easier to maintain; happy to revert if you disagree.